docs/apm/traces/working-with-tracing-data/tracing-dashboards.md

```diff
@@ -84,26 +81,73 @@ You are limited to three Trace List panels in a dashboard.
-| Number of errors | 0 | The number of errors in the trace. |
-| Status | 200 | The HTTP status code of the trace. A menu is available in this column when hovering on a row. The menu has an option to **Show similar traces**.<br/>![img](/img/traces/similar-traces-menu.png) |
+3. Configure a [Trace query](view-and-investigate-traces.md) to search for desired set of traces. 
+4. Select a time range or create a custom range for the panel. You can set this when creating or at any time when viewing the Dashboard panel.<br/> ![timerange.png](/img/traces/timerange.png)
+5. The **Chart Type** is set to Table.
+6. Enter the **Rows Per Page** for the panel, between 5 to 100. The default amount is 15. The panel automatically paginates traces to browse through and view all traces.
+7. Select the Table columns of trace data to load in the panel:
 
-1. Click the **General** tab to edit the Panel Details. Enter a name for the panel, set a **Title Font Size**, and add a short **Description**.<br/>  ![tracelist-details.png](/img/traces/tracelist-details.png)
-1. Click **Add to Dashboard**. The panel loads in your Dashboard to review all traces according to the query.<br/>  ![panel-trace.png](/img/traces/panel-trace.png)
+  | Column name | Example value | Description |
+  |:--|:--|:--|
+  | Trace ID | ffaf2f69ee8ad0c1 | The unique identifier of the trace. |
+  | Root Service | api | The service that started the trace. |
+  | Started At | 07/27/2020 09:01:04.533 | When the trace started. |
+  | Duration | 12.582 ms | The amount of time the trace spans.  |
+  | Number of spans | 35 | A trace consists of spans. This number tells you how many spans are in the trace. |
+  | Duration Breakdown | ![breakdown](/img/traces/breakdown.png) | Each color indicates a service. The colors assigned to services are always the same on your account. You can change the color in the span summary tab after clicking on the individual span in trace view.<br/>Hover over to view a percentage breakdown of how long each span covers in the trace.<br/>![img](/img/traces/span-hover-view.png) |
+  | Number of errors | 0 | The number of errors in the trace. |
+  | Status | 200 | The HTTP status code of the trace. A menu is available in this column when hovering on a row. The menu has an option to **Show similar traces**.<br/>![img](/img/traces/similar-traces-menu.png) |
+
+8. Click the **General** tab to edit the Panel Details. Enter a name for the panel, set a **Title Font Size**, and add a short **Description**.<br/>  ![tracelist-details.png](/img/traces/tracelist-details.png)
+9. Click **Add to Dashboard**. The panel loads in your Dashboard to review all traces according to the query.<br/>  ![panel-trace.png](/img/traces/panel-trace.png)
+
+
+### Dashboard Examples
```

Comment:
@JKashyap96 Moved these dashboards here from App Components. Does this look ok?

Reply:
Yes